repository/config/os.pan

```diff
@@ -1,39 +1,39 @@
-#######################################################################
-#
-# Standard repositories to use for the OS for YUM-based deployment
-#
-#######################################################################
-
-unique template repository/config/os;
-
-include { 'quattor/functions/repository' };
-
-@{
-desc =  if true, add epel-testing repository
-values = true or false
-default = false
-required = no
-}
-variable REPOSITORY_EPEL_TESTING_ENABLED ?= false;
-
-variable YUM_SNAPSHOT_NS ?= 'repository/snapshot';
-variable YUM_OS_SNAPSHOT_NS ?= YUM_SNAPSHOT_NS;
-
-# Ordered list of repository to load
-# NOTE: The repository which contains the AII rpms must be listed
-# first in the list.  If not, then AII installations will fail
-# because the post-install script will look in the wrong place.
-include { 'repository/config/quattor' };
-variable OS_REPOSITORY_LIST ?= {
-    append(OS_VERSION_PARAMS['version']+'_'+OS_VERSION_PARAMS['arch']);
-    append(OS_VERSION_PARAMS['version']+'_'+OS_VERSION_PARAMS['arch']+'_errata');
-    append(OS_VERSION_PARAMS['version']+'_'+OS_VERSION_PARAMS['arch']+'_updates');
-    append(OS_VERSION_PARAMS['major']+'_addons');
-    append(OS_VERSION_PARAMS['major']+'_epel');
-    if ( REPOSITORY_EPEL_TESTING_ENABLED ) {
-      append(OS_VERSION_PARAMS['major']+'_epel_testing');
-    };
-    append('site');
-};
-
-'/software/repositories' = add_repositories(OS_REPOSITORY_LIST,YUM_OS_SNAPSHOT_NS);
+#######################################################################
+#
+# Standard repositories to use for the OS for YUM-based deployment
+#
+#######################################################################
+ 
+unique template repository/config/os;
+ 
+include { 'quattor/functions/repository' };
+
+@{
+desc =  if true, add epel-testing repository
+values = true or false
+default = false
+required = no
+}
+variable REPOSITORY_EPEL_TESTING_ENABLED ?= false;
+
+variable YUM_SNAPSHOT_NS ?= 'repository/snapshot';
+variable YUM_OS_SNAPSHOT_NS ?= YUM_SNAPSHOT_NS;
+
+# Ordered list of repository to load

```

Comment:
Ha sorry! It was not clear in the way GitHub displayed things your comment was about  the comment and not the annotation of `REPOSITORY_EPEL_TESTING_ENABLED `.. at least I misunderstood it. Agreed, I'll convert it into an annotation... BTW, the comment is no longer true with YUM, repo list are no longer ordered and the first repo is not used for anything special...